.github/ISSUE_TEMPLATE/chore.yml

```diff
@@ -0,0 +1,23 @@
+---
+# https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema
+name: Chore
+description: Code maintenance improvements
+assignees: ["ferretdb-bot"]
+labels: ["code/chore"]
+body:
```

Comment:
I'd also add "why" field with something like "Why it it needs to be implemented?" so people can highlight the reason for such an improvement.

Reply:
#1160